src/main/java/de/retest/recheck/RecheckOptions.java

```diff
@@ -7,74 +7,160 @@
 import de.retest.recheck.ignore.CompoundFilter;
 import de.retest.recheck.ignore.Filter;
 import de.retest.recheck.ignore.RecheckIgnoreUtil;
+import de.retest.recheck.persistence.FileNamer;
 import lombok.AccessLevel;
 import lombok.AllArgsConstructor;
-import lombok.Getter;
 
+/**
+ * This class configures the behaviour of {@link Recheck} and their implementations.
+ */
 @AllArgsConstructor( access = AccessLevel.PRIVATE )
-@Getter
```

Comment:
IMHO you don't have to create getters manually to fix the warnings, I'm fine with them.

Reply:
I did it partly, too, because I was not happy with the JavaDoc between the fields and do not know if the maven plugin conforms with lombok.

Thoughts?

Reply:
Dunno, commenting getters somewhat feels like:

https://twitter.com/carterwickstrom/status/1014165500056596481

The code now contains a lot of noise due to the comments. But it's up to decide, you've put some effort in it.

Reply:
Mostly, yes (especially with `@return`). But for special getters (inside public API) as I implemented here, I think it is necessary.